test/kubemark/resources/hollow-node_template.yaml

```diff
@@ -117,8 +117,8 @@ spec:
           mountPath: /var/log
         resources:
           requests:
-            cpu: 20m
-            memory: 20Mi
+            cpu: {{npd_cpu_milli}}m
+            memory: {{npd_memory_mb}}Mi
```

Comment:
For consistency with hollow-proxy, can we change it:
(a) to be in Ki
(b) name it npd_mem
?

Reply:
Done, changed all to be `<component>_millicpu` and `<component>_mem_Ki`
I couldn't use `<component>_mem_ki` as `ki` is not the same as `Ki`. The former is 1000 while the latter is 1024.